### PR TITLE
Bookstore Project: Styling

### PR DESCRIPTION
**Pull request by Strangeal**
### Styling
So far it has been enjoyable😊 implementing all the necessary requirements for the project I learned so many things and it was also fun working on this project, what I did in the whole project :

- Added styling

### CODE REVIEW

File: src/components/Book.js [hunk 1]
```diff
@@ -10,14 +10,22 @@ const Book = () => {
   }, [dispatch]);
 
   const books = useSelector((state) => state.books);
+  console.log(books);
```
Comment (DammyShittu) [inline src/components/Book.js]: - [x] It is often not a good practice to push console logs of this nature as they can expose confidential information in more sophisticated apps

File: src/components/Book.js [hunk 2]
```diff
@@ -10,14 +10,22 @@ const Book = () => {
...
           <button type="submit" onClick={() => dispatch(deleteBook(itemId))}>
-            remove
+            |
+            {' '}
+            <span>Remove</span>
+            {' '}
+            |
           </button>
         </li>
       ))}
```
Comment (DammyShittu) [inline src/components/Book.js]: - [x] Kindly note that you are to replicate the design exactly. Ensure you add the other remaining styling as contained in the Zeplin design.

  Your Implementation | Design
  :---:|:---:
  ejur | fjfw

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @Strangeal,

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @Strangeal, 

### Good job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Strangeal** [comment]
Thanks for the approval

**Pull request merged**